src/like-service/app/Http/Controllers/LikeController.php

```diff
@@ -38,8 +40,11 @@ function doLike($request)
     function getLikeCountAndState($request)
     {
         $user_token = $request->cookie('jwt');
-        $postId = $request->header('postId');
+        $uri = $request->path();
+        $uriParts = explode('/', $uri);
+        $postId = $uriParts[2];
 
+        DB::enableQueryLog();
```

Comment:
[DISCUSS] Also here, I think we can remove the logging. Same for the other logging logic here? Or does the route not provide the value for {postId}?

Reply:
Logging logic removed